code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -47,6 +47,16 @@
 	reagents.add_reagent(NUTRIMENT, 3)
 	bitesize = 1
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/crabmeat
+	name = "crab meat"
+	desc = "Something killed the crab, and this is the result."
+	icon_state = "raw_crab"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/crabmeat/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 3)
+	bitesize = 1
```

Comment:
putting this in new() is weird, put it in the object definition

I know every other meat does it the way you're doing it, but as @PJB3005 once pointed out, there's no reason to maintain consistency with bad coding